src/components/Modal/Modal.tsx

```diff
@@ -19,6 +45,7 @@ export const Modal = (props) => {
     } = props;
 
     const { className: portalClass } = block({ className: portalClassName, ...rest });
+    const { className: bodyOpenClass } = elem('body--open', props);
```

Comment:
Here, if we'd need to have a class name `body--open`, that proper way to achieve it would be:
`elem('body', { ...props, open: true });`

But since [we don't need `open` modifier](https://github.com/textkernel/oneui/pull/377/files#diff-df00b1220859d4028b7de06ff6d065b2R2) at all, it can be just `elem('body', props);`